lms/djangoapps/courseware/rules.py

```diff
@@ -8,6 +8,7 @@
 from django.db.models import Q
 from opaque_keys.edx.keys import CourseKey
 from student.models import CourseEnrollment
+from student.role_helpers import has_staff_roles
 
 from .access import has_access
```

Comment:
this line is causing a circular dependency since course_duration_limits/models is called from has_access  
not sure which approach we want to take for this